lib/active_model/serializer/adapter/json_api/api_objects.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_model/serializer/adapter/json_api/api_objects/link'
+require 'active_model/serializer/adapter/json_api/api_objects/resource'
+require 'active_model/serializer/adapter/json_api/api_objects/resource_identifier'
+require 'active_model/serializer/adapter/json_api/api_objects/relationship'
+require 'active_model/serializer/adapter/json_api/api_objects/json_api_object'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module ApiObjects
```

Comment:
Could you elaborate on why autoloading would be preferable? (I must admit I lack knowledge about it, but it seemed to me that when you load `ApiObjects`, you always want the whole bundle, so might as well require everybody)